pkgs/apps/pysisyphus/default.nix

```diff
@@ -1,11 +1,22 @@
-{ fetchPypi, fetchFromGitHub, buildPythonPackage, lib, writeTextFile, writeScript, makeWrapper,
+{ fetchPypi, fetchFromGitHub, buildPythonPackage, lib, writeTextFile, writeScript, makeWrapper
+, pytestCheckHook
  # Python dependencies
- autograd, dask, distributed, h5py, jinja2, matplotlib, numpy, natsort, pytest, pyyaml, rmsd, scipy,
- sympy, scikit-learn, qcengine, ase, xtb-python, openbabel-bindings,
+, autograd, dask, distributed, h5py, jinja2, matplotlib, numpy, natsort, pytest, pyyaml, rmsd, scipy
+, sympy, scikit-learn, qcengine, ase, xtb-python, openbabel-bindings, pyscf
  # Runtime dependencies
- runtimeShell, jmol ? null, multiwfn ? null, xtb ? null, openmolcas ? null,
- pyscf ? null, psi4 ? null, wfoverlap ? null, nwchem ? null, orca ? null,
- turbomole ? null, gaussian ? null, gamess-us ? null, cfour ? null, molpro ? null,
+, runtimeShell
+, jmol, enableJmol ? false
```

Comment:
All QC packages are now enabled by default. Do you want to a few attributes with your best supported QC packages enabled in the top level file? E.g. `pysisyphus-molcas`, etc.?

Reply:
No they are actually not. All are disabled by default and lazy evaluation of Nix ensures they are never evaluated, unless the corresponding `enableX` attribute is `true` and they are all `false` by default. The `qchem.python3.pysisyphus`, that is available builds with the free quantum chemistry engines by default (XTB, NWChem, Molcas, and JMol and WFoverlap, which are also both free). The other engines are opt-in and the pysisyphus derivation that is exposed is useful for common work and can now be cached by Cachix and public Hydra.

Reply:
Sorry, I meant "disabled by default". I overlooked the `pythonPackages` entries.